client/state/data-layer/utils.js

```diff
@@ -38,6 +38,8 @@ export function convertKeysBy( obj, fn ) {
 }
 
 /**
+ * !!! There is TS version of this function: `import { convertSnakeCaseToCamelCase } from 'calypso/state/data-layer/convert-snake-case-to-camel-case';`
+ *
```

Comment:
First, we should probably use the `@deprecated` param. Second, do you think we could update linting to trigger a warning whenever someone tries to use that function?

Reply:
Added